facia-press/app/frontpress/FapiFrontPress.scala

```diff
@@ -124,15 +124,8 @@ trait FapiFrontPress extends QueryDefaults with Logging with ExecutionContexts {
   }
 
   private def getBackfill(collection: Collection): Response[List[PressedContent]] = {
-    collection
-      .collectionConfig
-      .apiQuery
-      .map { query =>
-        FAPI.backfill(query, collection, searchApiQuery, itemApiQuery)
-      }
-      .getOrElse {
-        Response.Right(Nil)
-      }.map(_.map(PressedContent.make))
+    FAPI.backfillFromConfig(collection, searchApiQuery, itemApiQuery)
```

Comment:
You should leave this name but use `collection.collectionConfig` instead; backfill definitely comes from config. You will have to change `facia-scala-client` and release it again.